source/ShipyardPanel.cpp

```diff
@@ -386,10 +399,24 @@ void ShipyardPanel::BuyShip(const string &name)
 
 
 
-void ShipyardPanel::SellShip()
+void ShipyardPanel::SellShipAndOutfits()
+{
+	SellShip(false);
+}
```

Comment:
honestly I'd just have sellship and default it to true (assuming you renamed toStorage to stockOutfits, otherwise false)
did you do this for clarity or bc the Dialog cant take an attribute (it does accept default ones tho no?)

Reply:
Dialog is given a function to call with no parameters, hence the two different functions for calling SellShip with different inputs.

Reply:
ok so default parameters dont work, good to know